README.md

```diff
@@ -4,7 +4,7 @@
 
 ## Introduction
 
-This project provides a working configuration of NGINX configured to act as 
```

Comment:
Please revert the whitespace changes in this commit.

Reply:
Reverted! It was an automatic VSCode thing